### PR TITLE
DASH-1220. Wallet model paranoid.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: client/src/redux/edge/actions.ts [hunk 1]
```diff
@@ -118,10 +118,13 @@ export const login = ({
       });
     }
 
+    console.log(account.archivedWalletIds);
```
Comment (trukhilio) [inline client/src/redux/edge/actions.ts:121]: Remove console

File: client/src/api/account.ts [hunk 2]
```diff
@@ -19,8 +19,12 @@ export default class Account extends Base {
 
   setWallets(
     fioWallets: FioWalletDoublet[],
+    archivedWalletIds?: string[],
   ): Promise<AccountSetWalletsResponse> {
-    return this.apiClient.post('account/wallets', { data: fioWallets });
+    return this.apiClient.post('account/wallets', {
+      data: fioWallets,
```
Comment (trukhilio) [inline client/src/api/account.ts:25]: Let's pass `archivedWalletIds` also in data

File: client/src/api/account.ts [hunk 3]
```diff
@@ -19,8 +19,12 @@ export default class Account extends Base {
 
   setWallets(
     fioWallets: FioWalletDoublet[],
+    archivedWalletIds?: string[],
```
Comment (trukhilio) [inline client/src/api/account.ts:22]: Let's receive params as an object ({ fioWallets, archivedWalletIds })

**Review by trukhilio — COMMENTED** [review]
Check comments

File: client/src/redux/edge/actions.ts (hunk 1 above)
Comment (andreyvEze) [reply client/src/redux/edge/actions.ts]: done

File: client/src/api/account.ts (hunk 2 above)
Comment (andreyvEze) [reply client/src/api/account.ts]: done

File: client/src/api/account.ts (hunk 3 above)
Comment (andreyvEze) [reply client/src/api/account.ts]: done

**Review by trukhilio — APPROVED** [review]

**Pull request merged**